sys-apps/nawk/nawk-20121220-r3.ebuild

```diff
@@ -1,13 +1,13 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
 inherit toolchain-funcs
 
 DESCRIPTION="Brian Kernighan's pattern scanning and processing language"
-HOMEPAGE="http://cm.bell-labs.com/cm/cs/awkbook/index.html"
-SRC_URI="mirror://gentoo/${P}.tar.gz"
+HOMEPAGE="https://www.cs.princeton.edu/~bwk/btl.mirror/"
+SRC_URI="https://www.cs.princeton.edu/~bwk/btl.mirror/awk.tar.gz"
```

Comment:
Please keep the current `SRC_URI`, `awk.tar.gz` is unversioned and that leads to problems with mirroring.